crates/sui-graphql-rpc/src/types/transaction_block_kind/consensus_commit_prologue.rs

```diff
@@ -26,6 +28,7 @@ pub(crate) struct ConsensusCommitPrologueTransaction {
     consensus_commit_digest: Option<ConsensusCommitDigest>,
     /// The checkpoint sequence number this was viewed at.
     checkpoint_viewed_at: u64,
+    consensus_determined_version_assignment: Option<ConsensusDeterminedVersionAssignment>,
```

Comment:
Could you update the doc comment to explain what is this field used for?

Reply:
Done. Thanks for taking a look @stefan-mysten !

Regarding graphql tests, the graphql-e2e-tests currently using protocol version 39, and we need to update it to protocol version 46 in order to use consensus commit prologue V3. I don't know if there are folks on the team who regularly bring it up to date. If not, I can try to update it.

Reply:
@halfprice thanks for fixing that. 
Regarding the protocol update, indeed we don't touch it often. @amnn or @emmazzz might know better, so I'll let them suggest what to do here.

Reply:
Hi @halfprice, indeed if you could bring them up-to-date, that would be appreciated -- we only do it as and when it is needed to avoid churn. To save the blame for your PR, I would recommend doing that in a separate PR.

Reply:
Hi @stefan-mysten @amnn , created #17819 . PTAL!